Lesson1/Dima_Gnatyk/tests.cpp

```diff
@@ -0,0 +1,19 @@
+#include "main.cpp"
+#include <gtest/gtest.h>
+
+template <class T>
+std::vector<T> concat(const std::vector<T> &data_vector_1, const std::vector<T> &data_vector_2);
+
+namespace {
+class TestConcat : public ::testing::Test{
+
+};
+TEST_F(TestConcat, Concat_Vector){
```

Comment:
Is it still work in progress?
But I see gtest is set up for this project, it is good.

Reply:
There should be a 5 tests for each test suits (one for one task - 3 in summary).